packages/react-native-editor/src/setup.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Internal dependencies
+ */
+import initialHtml from './initial-html';
+import setupApiFetch from './api-fetch-setup';
+
+/**
+ * External dependencies
+ */
+import { I18nManager } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	validateThemeColors,
+	validateThemeGradients,
+} from '@wordpress/block-editor';
+import { unregisterBlockType, getBlockType } from '@wordpress/blocks';
+import { addAction, addFilter } from '@wordpress/hooks';
+import * as wpData from '@wordpress/data';
+import { initializeEditor } from '@wordpress/edit-post';
+import { registerCoreBlocks } from '@wordpress/block-library';
+
+let editorComponent;
+let blocksRegistered = false;
```

Comment:
I would posit it would be clearer to place these decorations closer to their reference sites. E.g. place `let editorComponent;` just before the default explore. WDYT?

Reply:
I'm accustomed to placing variable/constant definitions before functions but I don't have a strong preference for this so I can move them to the functions where they're used.

Reply:
I applied this suggestion in https://github.com/WordPress/gutenberg/pull/37073/commits/4841a8bf06b25a64fe1ce8fa64b763821e142a0e.

Reply:
> I'm accustomed to placing variable/constant definitions before functions but I don't have a strong preference for this so I can move them to the functions where they're used.

Understood. Feel free to revert https://github.com/WordPress/gutenberg/commit/4841a8bf06b25a64fe1ce8fa64b763821e142a0e if you prefer. My inquiry was merely an inquiry, and not an explicit request. 

I do generally place "global" constants at the top of files, e.g. `const API_KEY = '';`. However, for most everything else I generally try to keep related code together to reduce the amount of line jumping needed to comprehend a portion of code.